src/Components/Facility/FacilityHome.tsx

```diff
@@ -302,6 +302,21 @@ export const FacilityHome = (props: any) => {
           <CoverImage />
           {editCoverImageTooltip}
         </div>
+      ) : (
+        <div
+          className={`group md:hidden flex w-full self-stretch shrink-0 bg-gray-300 items-center justify-center relative z-0 ${
+            hasPermissionToEditCoverImage && "cursor-pointer"
+          }`}
```

Comment:
Use `clsx`

Reply:
Hey @developedBySJ Gigin has told no to use clsx totally. If we show the pencil icon by default in Mobile Devices the cover image would not be visible in mobile devices.

Reply:
@patelaryan7751 can you refactor it using classNames _(should be available in `src/Utils/utils.ts`)_